qt/widgets/common/src/FitPropertyBrowser.cpp

```diff
@@ -2211,7 +2211,7 @@ void FitPropertyBrowser::addConstraint(int f, bool lo, bool up) {
   if (!h)
     return;
 
-  double x = m_doubleManager->value(parProp);
+  double x = parProp->valueText().toDouble();
```

Comment:
Is this the best approach in this circumstance? There's no guarantee that the text representation of a property will match up to the associated value (in this case a double). If the representation were to change in the future this could break. The purpose of the managers is to be able to retrieve the associated values.

The m_parameterManager field of the FitPropertyBrowser should contain the parameter value (not the double manager). If it cannot be found here, this would be a further underlying issue.

Reply:
I believe it cannot be found there. I confirmed with David before he left when I was working on it.